src/jira-release.js

```diff
@@ -10,9 +10,7 @@ async function run() {
 
     let jiraVersionName = `${context.repo.repo}-${tag_name.replace(/^v/, '')}`
 
-    const {
-      body: { data },
-    } = await jiraClient
+    const data = await jiraClient
```

Comment:
@DominicCabral , don't we need to destructure the data value still? I.e. 
```suggestion
    const { data } = await jiraClient
```
Looks like it is still the [case](https://github.com/sindresorhus/got), unless I am missing something?

Reply:
Probably docs only dereference data because that's how API returns it.